### PR TITLE
Create Taipy Cloud deployment documentation

### LINKED ISSUE #816: Add taipy-cloud deployment doc
Deploying an application on taipy-cloud must be documented.
We already have content and videos that can be added to the current documentation.

### PR DESCRIPTION
**Pull request by FlorianJacta**
Resolves: #816 

Move the Taipy Cloud tip to the Manuals with extra info.

image

image

image

image

image

image

[Deploying on Taipy Cloud - Taipy.zip](https://github.com/Avaiga/taipy-doc/files/14241244/Deploying.on.Taipy.Cloud.-.Taipy.zip)

### CODE REVIEW

**Review by jrobinAV — APPROVED** [review]

File: docs/manuals/run-deploy/deploy/taipy-cloud/index.md [hunk 1]
```diff
@@ -1,4 +1,151 @@
...
+    Using Windows-specific libraries is not supported, as the machines operate 
+    on a Linux environment.
+
+
+!!! Note
+    For uploading large data sets or sizable files, please use the `/home/user/data`
+     directory. This location preserves your data across deployments. To transfer your 
+     files, establish a connection to your machine using 
+     [SSH or Filezilla](#connect-with-ssh-or-filezilla) and upload your data 
+     directly to this directory.
```
Comment (florian-vuillemot) [inline docs/manuals/run-deploy/deploy/taipy-cloud/index.md]: `/home/user/data` will not be supported in the feature due to the new data management on taipy-cloud. So please do not document it.

File: docs/manuals/run-deploy/deploy/taipy-cloud/index.md [hunk 2]
```diff
@@ -1,4 +1,151 @@
...
+   - **Password**: Enter the  password you have provided for the machine.
+
+**Click Quickconnect**: If everything is configured correctly, FileZilla will establish a 
+secure connection to your Taipy Cloud machine, and you'll be able to start transferring 
+files.
+
+### Best Practices for File Transfer
+
+- **Organize Your Files**: Keep your data organized in the `/home/user/data` directory to 
+ensure persistence across deployments.
```
Comment (florian-vuillemot) [inline docs/manuals/run-deploy/deploy/taipy-cloud/index.md]: Same

File: docs/manuals/run-deploy/deploy/taipy-cloud/index.md [hunk 3]
```diff
@@ -1,4 +1,151 @@
...
+
+- **Manage File Sizes**: To optimize performance and deployment times, avoid uploading 
+unnecessary large files. Use compression when possible.
```
Comment (florian-vuillemot) [inline docs/manuals/run-deploy/deploy/taipy-cloud/index.md]: Including `__pycache__`, `virtualenv` and `git` files.

File: docs/manuals/run-deploy/deploy/taipy-cloud/index.md [hunk 4]
```diff
@@ -1,4 +1,151 @@
...
+
+- **Security**: Always ensure your SSH keys are secure and not shared with unauthorized 
+individuals.
```
Comment (florian-vuillemot) [inline docs/manuals/run-deploy/deploy/taipy-cloud/index.md]: We do not provide SSH keys, only SSH password.

**Pull request merged**